### DISCUSSION TITLE
Laser is burning at full strength on some parts of my engraving

### DISCUSSION THREAD

**Discussion by MageDK** [arkypita/LaserGRBL#1677] [Q&A]
I have a png image that I am using to laser engrave onto a piece of leather.  I have tried different laser strength settings and no matter what I set it to there are certain parts of the pattern that burn at 100% and its the same spots over and over again.  

What am I doing wrong?

Here is where I am setting the laser strength 
![image](https://user-images.githubusercontent.com/3934561/146426446-ba0a7f97-240a-4819-a62c-f1ca06bfbad9.png)

Here is the leather I am engraving.  I have one job set at 5%, one at 15% and one at 50%.  You can see on all 3 that the same spot on each is getting charred.
![leather-burn](https://user-images.githubusercontent.com/3934561/146426805-fbe15b16-e7cf-4827-9279-79c65890a58e.jpg)
right at the beginning, the tips of 2 letters and at the end.  Same spots over and over again.

**Comment by arkypita** [comment]
Having SMin and SMax the same values is wrong. Use SMin = 0 and SMax something between 30% and 60%
https://lasergrbl.com/usage/raster-image-import/target-image-size-and-laser-options/

M4 is better for engraving.

**Reply by MageDK** [reply]
What's the difference between M3 and M4?

**Reply by arkypita** [reply]
I give you a link, did you read it?
